### PR TITLE
added some bash shell commands

### PR DESCRIPTION
**Pull request by paberline**
This is my first attempt at using talon for editing code. I decided to add some bash shell commands to terminal.py to ease hands free operation using talon.@dwiel Please review and comment

### CODE REVIEW

File: apps/terminal.py [hunk 1]
```diff
@@ -75,6 +75,10 @@ def ssh_copy_id_servers(m):
 
 
 keymap = {
+    "shell Whereami": "pwd ",
+    "shell home": "~/",
+    "shell changedir": "cd",
+    "shell copy": "cp",
```
Comment (zach-nervana) [inline apps/terminal.py]: this is already implemented in this file on line 124, unless you are intending different semantics

File: apps/terminal.py [hunk 2]
```diff
@@ -75,6 +75,10 @@ def ssh_copy_id_servers(m):
 
 
 keymap = {
+    "shell Whereami": "pwd ",
+    "shell home": "~/",
+    "shell changedir": "cd",
```
Comment (zach-nervana) [inline apps/terminal.py]: this is already implemented on line 91, unless you are intending different semantics

File: apps/terminal.py (hunk 1 above)
Comment (paberline) [reply apps/terminal.py]: you are correct. sorry, i'll remove it from the PR

File: apps/terminal.py (hunk 2 above)
Comment (paberline) [reply apps/terminal.py]: yes i see cd is implemented already. i'l remove it from the PR. i can't see pwd or ''~/"  anywhere. can I leave that in?

**Pull request merged**